validator/accounts/wallet/wallet.go

```diff
@@ -27,8 +27,6 @@ import (
 var log = logrus.WithField("prefix", "wallet")
 
 const (
-	// KeymanagerConfigFileName for the keymanager used by the wallet: imported, derived, or remote.
-	KeymanagerConfigFileName = "keymanageropts.json"
```

Comment:
I had to move this to `validator/keymanager/keymanager.go` or I was running into circular dependencies.
Any reasons why this variable would rather be in `wallet` than `keymanager`?

Reply:
keymanager is good